app/models/annotation/nml/NmlParser.scala

```diff
@@ -140,7 +140,7 @@ object NmlParser extends LazyLogging with ProtoGeometryImplicits {
   }
 
   private def parseEditRotation(node: NodeSeq) = {
-    node.headOption.flatMap(parseRotation)
+    node.headOption.flatMap(parseRotationForParams)
```

Comment:
I guess this fixes a bug you found with the tests?

Reply:
Yes it is because of difference in parsing `EditRotation` of `nodes` and the general param